lib/letter_avatar/configuration.rb

```diff
@@ -8,6 +8,14 @@ def cache_base_path=(v)
       @cache_base_path = v
     end
 
+    def fill_color
+      @fill_color
```

Comment:
could you please change this getter, let it always return a walue (`FILL_COLOR` by default) - like eg. `colors_palette` below

and then, in the `Avatar` use only `LetterAvatar.fill_color` since it'll return the default :-)